ui/css/perf.css

```diff
@@ -219,6 +219,39 @@ to { opacity: 0; }
     overflow: hidden;
 }
 
+/*Test Chooser graph*/
```

Comment:
Not to nit too much, but perhaps should be a space like other comments in the file. If a dev changes it later, they are less likely to mangle the comments between the `/*` markers

`/* Test Chooser graph */`